agnoster.zsh-theme

```diff
@@ -82,7 +82,7 @@ prompt_context() {
 prompt_git() {
   local color ref
   is_dirty() {
-    test -n "$(git status --porcelain --ignore-submodules)"
+    test -n "$(git status --porcelain --ignore-submodules &> /dev/null)"
```

Comment:
Per @belak, should this be `2> /dev/null` instead? `&>` will redirect stdout as well.